### ISSUE TITLE
Resource documentation example syntax is incorrect

### ISSUE THREAD

**Issue by Invincibear** [sematext/terraform-provider-sematext#57]
The resource documentation template is generating incorrect Terraform syntax for your provider. For example, from https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_app_infra
```
resource "sematext_app_infra" "myapp" {
  name = "my app name"
  billing_plan_id = <[plan id](../guides/plans.md)>
  apptoken {
    name "my apptoken name"
    create_missing true
  }
}
```

* The parameters inside `apptoken {}` are missing `=` signs
* `create_missing` is not a valid parameter
* `name` should be `names`
* `names` requires a list of strings
* The reference to `plans.md` is broken (probably because it is inside a code block and isn't interpreted)

**Comment by hollerloudly** [comment]
Thanks for PR - applied, released in v0.6.2

**Comment by hollerloudly** [comment]
Closing.
